templates/templates/base.html

```diff
@@ -113,7 +113,7 @@
       <div class="u-fixed-width">
         <div class="p-notification--positive u-no-margin--bottom">
           <div class="p-notification__content">
-            <p class="p-notification__message">Thank you for signing up for our newsletter!<br/>In these regular emails you will find the latest updates about Ubuntu and upcoming events where you can meet our team.<a href="#" onclick="location.href = document.referrer; return false;"><i class="p-notification__close">Close</i></a></p>
```

Comment:
@lizzochek when you are back, can you check what removing this could break? Thanks, let me know if you need help with a better solution. I don' t think applying this globally is good, maybe we should encpasulate it within some function so that it triggers only in that specific situation and also it'd be easier to track where this code is used.